README.md

```diff
@@ -1,2 +1,56 @@
 # sustainable-society
 Collection of documents to use when needing an advice regarding sustainability 
+
+## Food
+-  Buy local and seasonal food, no need to buy summer food in winter, or if you do have cravings for it buy (or do it yourself) pickled or in naother way preserved version
+
+- Avoid supermarkets for most of the things, buy there only the things that are not available locally
+
+- Eat less meat, it's more sustainable and healthier
+
+## Clothing
```

Comment:
I would also add: 

> Always look for the `About us`, `Ethical manifesto` or `Our values` page before you buy from any brand. See what their promise is, then you can research whether or not they stand behind it.

> Try to repair or repurpose clothes before throwing it away

Greenwashing is usually more abstract and sends very broad messages like "we care for the environment" without explaining why.